docs/reference/dapis/understand/index.md

```diff
@@ -25,12 +25,14 @@ such as the latest cryptocurrency, stock, and commodity prices. They can power
 various decentralized applications such as DeFi lending, synthetic assets,
 stable coins, derivatives, NFTs, and more.
 
-## Data feeds values stored on-chain
+## Values stored on-chain
 
-API providers, owners of first-party Airnodes, store data feed values on-chain
-as individual beacons that in are turn sourced by the
+API providers, owners of first-party Airnodes, provide the signed data used to
+store beacon values (via AirSeeker) on-chain as individual beacons. A dAPI's
```

Comment:
"as individual beacons" seems redundant with "store beacon values" in this sentence. You could combine by having "...used to store individual beacon values on-chain."

Reply:
Done and thanks